### ISSUE TITLE
Gestures Can not be Detected on Hololens2

### ISSUE THREAD

**Issue by keli95566** [microsoft/MixedRealityToolkit-Unity#10226]
## Describe the bug

Gesture not detected on hololens2 using IMixedRealityGestureHandler<Vector3>

## To reproduce

1. Copy the ```GestureTester``` Gameobject from the MRTK example for the gesture event scene. 
2. Paste the Gameobject in a new scene in the same project.
3. The above-mentioned things work. But I implemented almost the SAME code with a new game object and it does not work. The code is as below. It is expected to have the same log but the log is triggered. No event functions were called. And no errors were given. Can someone please help?

```csharp
using System.Collections;
using System.Collections.Generic;
using Microsoft.MixedReality.Toolkit;
using Microsoft.MixedReality.Toolkit.Input;
using Microsoft.MixedReality.Toolkit.Utilities;
using UnityEngine;


public class FingerTipController : MonoBehaviour, IMixedRealityGestureHandler<Vector3>
    {

        private IMixedRealityHandJointService handJointService = null;
        private IMixedRealityDataProviderAccess dataProviderAccess = null;

        void Start()
        {
            handJointService = CoreServices.GetInputSystemDataProvider<IMixedRealityHandJointService>();

            if (handJointService == null)
            {
                Debug.LogError("Can't get IMixedRealityHandJointService.");
                return;
            }

            dataProviderAccess = CoreServices.InputSystem as IMixedRealityDataProviderAccess;
            if (dataProviderAccess == null)
            {
                Debug.LogError("Can't get IMixedRealityDataProviderAccess.");
                return;
            }



        }

        void Update()
        {

        }

        public void OnGestureStarted(InputEventData eventData)
        {
            Debug.Log($"OnGestureStarted [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");

            MixedRealityInputAction action = eventData.MixedRealityInputAction;
            Debug.Log(action);
        }


        public void OnGestureUpdated(InputEventData eventData)
        {
            Debug.Log($"OnGestureUpdated [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");

            MixedRealityInputAction action = eventData.MixedRealityInputAction;

        }

        public void OnGestureUpdated(InputEventData<Vector3> eventData)
        {
            Debug.Log($"OnGestureUpdated [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");

            MixedRealityInputAction action = eventData.MixedRealityInputAction;

        }
        public void OnGestureCompleted(InputEventData<Vector3> eventData)
        {
            Debug.Log($"OnGestureCompleted [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");


        }

        public void OnGestureCompleted(InputEventData eventData)
        {
            Debug.Log($"OnGestureCompleted [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");


        }

        public void OnGestureCanceled(InputEventData eventData)
        {
            Debug.Log($"OnGestureCanceled [{Time.frameCount}]: {eventData.MixedRealityInputAction.Description}");


        }

}


```
## Your setup (please complete the following information)

- Unity Version [e.g. 2020.3.161f]
- MRTK Version [e.g. v2.6]

## Target platform (please complete the following information)

- HoloLens 2

**Comment by keveleigh** [comment]
Are you using Windows XR Plugin? If so, unfortunately we won't have support for gestures due to the way it was implemented on the Unity side, and it's unlikely that they'll add the feature we depend on at this point.

If you're using Mixed Reality OpenXR Plugin, gestures shipped in MRTK 2.7.0 and Mixed Reality OpenXR Plugin 0.9.5. It looks like we may have recently had a regression if you've updated to Unity's OpenXR Plugin 1.5.0, but we're working on a fix for that in the next release.

I'm going to close this issue out, as it's a combination of:
1. Should be all good on legacy XR
1. Won't work with Windows XR Plugin and there's not really anything we can do at this point
1. Should be all good on OpenXR, other than the regression when using Unity's 1.5.0 or newer plugin.

Please reopen if any of the above doesn't match your observations!

**Comment by keveleigh** [comment]
Okay @keli95566 I took a deeper look:

I confirmed the repro with the latest versions of MR OpenXR and Unity's OpenXR (1.6.0 for both).
I've also confirmed that it is **fixed** in the upcoming 1.7.0 release of MR OpenXR.
I can also confirm that I was **not** able to repro on MR OpenXR 1.5.0 and Unity's OpenXR 1.4.2. It appears specific to Unity's 1.5.0+ releases, which MR OpenXR takes as a dependency in 1.6.0.

All tests were run on-device using the HandInteractionGestureEventsExample in MRTK2. Gestures do not work over remoting for OpenXR.
